examples/asyncio/stable_diffusion_run.py

```diff
@@ -0,0 +1,16 @@
+import asyncio
+
+from pipeline.api.asyncio import PipelineCloud
+
+api = PipelineCloud()
+
+
+async def run():
+    res = await api.run_pipeline(
+        "pipeline_67d9d8ec36d54c148c70df1f404b0369", [["a dog"], {}]
```

Comment:
might be better to use a pipeline alias here?

Reply:
How do I do that?

Reply:
These need to be setup on the backend but you should be able to pass in an alias instead of the pipeline id. I'm not sure if any have been set up yet so might be best to check with @alexpearce

Reply:
Okay, I think resolving this in a separate PR where we bring all of the examples forward to the new alias system might be best?

Reply:
yeah can do